### DISCUSSION TITLE
Will MAUI be production ready on Linux?

### DISCUSSION THREAD

**Discussion by UltraBurstXD** [dotnet/maui#326] [Q&A]
Hi,
I realized that project Xamarin didn't offered any kind of support for Linux for long time, so as the evolution of Xamarin I'm wondering if there will be any notable support?

**Comment by Rand-Random** [comment]
Only if the Community does give support.
You can look here to see what Microsoft will support itself
https://github.com/dotnet/maui#xamarinforms-vs-net-maui

**Comment by knuxbbs** [comment]
I hope that, at least, I can run an Android app on my Linux machine.

**Comment by knocte** [comment]
> I hope that, at least, I can run an Android app on my Linux machine.

You mean via Anbox? That wouldn't have any relation to Maui though.
